components/script/script_thread.rs

```diff
@@ -1581,30 +1579,11 @@ impl ScriptThread {
 
     fn collect_reports(&self, reports_chan: ReportsChan) {
         let mut path_seg = String::from("url(");
-        let mut dom_tree_size = 0;
         let mut reports = vec![];
```

Comment:
This is unused now.

Reply:
I am interpreting this to mean that since we don't add any dom-tree entries to the reports vector anymore we can use the return of get_reports directly. ie. `reports_chan.send(get_reports(self.get_cx(), path_seg));` Is that what you meant?

Reply:
Oh, my mistake. I was skimming the changes and didn't look closely enough. You can ignore this comment.